projects/plugins/crm/.phpcs.dir.xml

```diff
@@ -1,5 +1,25 @@
 <?xml version="1.0"?>
 <ruleset>
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+			<property name="text_domain" type="array">
+				<element value="zero-bs-crm" />
+			</property>
+		</properties>
+	</rule>
+	<rule ref="Jetpack.Functions.I18n">
+		<properties>
+			<property name="text_domain" value="zero-bs-crm" />
+		</properties>
+	</rule>
+
+	<rule ref="WordPress.Utils.I18nTextDomainFixer">
+		<properties>
+			<property name="old_text_domain" type="array" />
+			<property name="new_text_domain" value="zero-bs-crm" />
+		</properties>
+	</rule>
```

Comment:
I tried to satisfy PHPCS but it's a can of worms 😂 I just stopped after this one. We'll have to merge it with the failing tests.

Reply:
It doesn't seem that bad to me. The additions above are good to start.

I see the following other sniffs being reported on this PR:

* `Generic.WhiteSpace.ScopeIndent.IncorrectExact` - Looks like this could be fixed easily enough by indenting the function correctly. The file will be inconsistent because you have [almost](https://github.com/Automattic/jetpack/blob/cee249591c7134cb78fc19e62e1cfee0ff2a59e5/projects/plugins/crm/includes/ZeroBSCRM.SystemChecks.php#L437-L444) everything else in the file oddly indented, but at some point you'll have to fix that anyway (but I'd recommend doing that in a separate PR).
* `WordPress.Arrays.CommaAfterArrayItem.NoComma` - Easy to fix, just add the missing comma.
* `WordPress.WP.I18n.MissingTranslatorsComment` - Easy to fix, just add the missing comment..
* `WordPress.NamingConventions.ValidFunctionName.FunctionNameInvalid` - If you want to ignore this one, either with a `phpcs:disable` in ZeroBSCRM.SystemChecks.php or for your entire plugin here, I won't try to stop you. Personally I don't care for WordPress's naming conventions either, but it's the standard we've chosen to follow for Jetpack-the-plugin itself so it's part of the ruleset.

The "Check linter exclude lists" is also easy to fix, just delete the flagged line from that file.

Reply:
I've fixed most of the linting issues but the "can of worms" I was mostly worried about remains (renaming `zeroBSCRM_checkSystemFeat`).

I feel like any more changes in this PR will make it too bloated as well